### PR TITLE
Added data for Fetch API

### PR DESCRIPTION
**Pull request by lmenus**
Found the main [Fetch API page](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) using the old format table. The linter was telling me `firefox` should be an object, but I looked at other files and it's often defined as an array?

Apart from that, I hope I did not mess up the commits! It shows me 3 commits made, but 1 should be already merged with the main branch, and 2 are the same as I merged this in development from a separate branch – what went wrong?

PS. First code commit, easy on me please. 😬

### CODE REVIEW

**Comment by lmenus** [comment]
Hi @Elchi3, here's the promised commit, did I mess up the commits?

File: api/Fetch.json [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.fetch.enabled"
+                }
+              ]
+            },
+            {
+              "version_added": "52",
+              "note": "Prior to Firefox 52, <code>get()</code> only returned the first value in the specified header, with <code>getAll()</code> returning all values. From 52 onwards, <code>get()</code> now returns all values and <code>getAll()</code> has been removed."
```
Comment (maboa) [inline api/Fetch.json]: Should be `notes` not `note` :)

**Comment by Elchi3** [comment]
Hi @lmenus! Thanks for you first PR, glad you're getting into this!

Some things to note:
- You likely did not branch this new work from an fresh master branch, but probably from the feature branch you created for your other PR? It's often best to start branching new work from an updated master branch. So first checkout master, then pull the latest changes into master, then branching a new feature branch.
- As Mark found out correctly, `note` should be `notes`. Sorry about the unhelpful error message and sorry `notes` does accept a string and not just arrays. We have an issue about that filed.
- The page you are trying to convert to here is a "landing" or "overview" page about the Fetch API. The Fetch API consists of many features and we haven't agreed on a good structure for how to put in "collection features" into this data set yet. That said, I would encourage you to pick something else for the moment. Usually there are a few "help wanted" issues in the issues list, but there are so many open PRs right now, that I've not added new ones for now to stop the fire hose for the maintainers a bit. If you like, you can help migrating by picking an API interface from this list https://docs.google.com/spreadsheets/d/1ivgyPBr9Lj3Wvj5kyndT1rgGbX-pGggrxuMtrgcOmjM/edit#gid=926663640 (where there is no name/assignee yet).

Thanks again for your contributions! :100:

File: api/Fetch.json [hunk 2]
```diff
@@ -0,0 +1,157 @@
+{
+  "api": {
+    "Fetch": {
```
[the remark this answers is not in the document]
Comment (lmenus) [reply api/Fetch.json]: I was thinking about this for a while, will fix.

File: api/Fetch.json (hunk 1 above)
Comment (lmenus) [reply api/Fetch.json]: Ah yes, thank you!

**Comment by lmenus** [comment]
Thank you @Elchi3, will do! It's true I probably didn't pick the best one to begin with, I even added a new row for this in the Spreadsheets file, just letting you know in case it should be deleted.

File: api/Fetch.json [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+              }
+            ],
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Fetch.json]: Both versions of Opera should say `"30"`.

Comment (lmenus) [reply api/Fetch.json]: @jpmedley Thank you.

**Comment by lmenus** [comment]
@jpmedley Fixed.
@Elchi3 Commits cleaned up.
@maboa Left `Fetch` unchanged for now until somebody else chips in with how this should be organised.

**Comment by Elchi3** [comment]
Thanks again for your work! As said above, we're currently not migrating "overview pages" into this repo. "Fetch" is not an interface and this folder only contains interfaces for now: https://github.com/mdn/browser-compat-data/tree/master/api

I see you've picked some other pages in new PRs, that's cool! 
I'm going to close this one unmerged for now, though. Thanks for your understanding.

**Pull request closed without merging**